app/derivative_services/av_derivative_service.rb

```diff
@@ -69,9 +69,23 @@ def generate_hls_playlist(output, dir, built_files)
     end
   end
 
+  # rubocop:disable Metrics/MethodLength
   def generate_hls_derivatives(dir)
     _stdout, _stderr, status =
-      Open3.capture3("ffmpeg", "-y", "-i", file_object.disk_path.to_s, "-hls_list_size", "0", "-hls_time", "10", "-f", "hls", "-codec:a", "libmp3lame", "-muxdelay", "0", dir.join("hls.m3u8").to_s)
+      Open3.capture3("ffmpeg", "-y",
+                     "-i", file_object.disk_path.to_s,
+                     "-f", "hls", # HTTP Live Streaming output format
+                     "-hls_list_size", "0", # playlist will contain all entries
+                     "-hls_time", "10", # segments are 10s in length
+                     "-c:v", "libx264", # encode video with H.264
+                     "-preset", "slow", # slow encoding for better compression
+                     "-crf", "20", # video quality from 0-51
+                     "-vf", "format=yuv420p", # needed for Firefox. See: https://trac.ffmpeg.org/wiki/Encode/H.264#Encodingfordumbplayers
+                     "-movflags", "+faststart", # good option for web video
+                     "-c:a", "aac", # encode audio with AAC
```

Comment:
I love all these new comments so much.

Did you try it with the previous audio codec again once you got it working? I'm not sure why we had that one to begin with so I'm wondering if we should default to keeping it.

Reply:
I didn't. However, I think these are correct ("most compatible") settings for video. Seems to work for audio as well. I wonder if the mp3 codec might be the source of some of the audio issues we're seeing in new versions of UV. We could wait for Trey to return and provide some context.

Reply:
I just copied something from the internet, AAC seems fine.